research/src/utils/typography.js

```diff
@@ -22,7 +22,14 @@ const typography = new Typography({
   headerGrayHue: 'warm',
 
   // Body
-  bodyFontFamily: ['Source Sans Pro', 'Georgia', 'serif'],
+  bodyFontFamily: [
+    'Source Sans Pro',
+    'Helvetica Neue',
```

Comment:
Why change this from Georgia (a common system font), to Helvetica Neue (non-system, typically paid font)

Reply:
I just copied the font family from above (`headerFontFamily`) so the addition of a paid font wasn't intentional 😅. I don't know too much about fonts so happy to adjust.

My only intention was to remove serif fonts from this family since the final font we render is a sans-serif font.